account_reversal/account_reversal.py

```diff
@@ -60,7 +60,9 @@ def _move_reversal(self, reversal_date,
         self.ensure_one()
         period_obj = self.env['account.period']
 
-        if not reversal_period_id:
+        if reversal_period_id:
+            reversal_period_id = period_obj.browse([reversal_period_id])[0]
```

Comment:
To be consistent with variable naming, this should be reversal_period (without id).